CHANGES.md

```diff
@@ -5,7 +5,7 @@ Bugfixes
 --------
 
 - Fix regression in 1.1rc1 where OPTIONS requests to the media repo would fail. ([\#5593](https://github.com/matrix-org/synapse/issues/5593))
-- Removed the `SYNAPSE_SMTP_*` docker container environment variables. Using these environment variables prevented the docker container from starting in Synapse v1.0, even though they didn't actually allow any functionality anyway. Users are advised to remove `SYNAPSE_SMTP_HOST`, `SYNAPSE_SMTP_PORT`, `SYNAPSE_SMTP_USER`, `SYNAPSE_SMTP_PASSWORD` and `SYNAPSE_SMTP_FROM` environment variables from their docker run commands. ([\#5596](https://github.com/matrix-org/synapse/issues/5596))
+- Removed the `SYNAPSE_SMTP_*` docker container environment variables. Using these environment variables prevented the docker container from starting in Synapse v1.0, even though they didn't actually allow any functionality anyway. Users are advised to only use the `SYNAPSE_CONFIG_*` environment variables for configuration from their docker run commands. See the [documentation](https://github.com/matrix-org/synapse/tree/develop/docker) for more information. ([\#5596](https://github.com/matrix-org/synapse/issues/5596))
```

Comment:
I'm not sure that "the `SYNAPSE_CONFIG_*` environment variables" is a clear way to characterise it - I would think of it more as "providing a configuration file".

Also, if you're going to link to documetnation from the changelog, I suggest you use the branch name rather than `develop` which might change in the future.

To be honest though, I would remove everything after "they didn't actually allow any functionality anyway": it's up to other changelog entries (or maybe a summary) to document the switch to config files.